Source/WebCore/contentextensions/ContentExtensionsBackend.cpp

```diff
@@ -235,6 +235,7 @@ ContentRuleListResults ContentExtensionsBackend::processContentRuleListsForLoad(
     URL mainDocumentURL;
     URL frameURL;
     bool mainFrameContext = false;
+    RequestMethod requestMethod = readRequestMethod(initiatingDocumentLoader.request().httpMethod().convertToASCIILowercase()).value();
```

Comment:
.value() will crash if it's an unexpected value.  Let's use .value_or(RequestMethod::None) instead to be more robust against unexpected entries.

Also, convertToASCIILowercase does a little bit of unnecessary work.  What would you think about using equalIgnoringASCIICase in readRequestMethod instead?  It would make the parsing a little more flexible, but it's also pretty common to have all caps request methods.